### PR TITLE
Bug/twitter bot reply timer

### PR DESCRIPTION
**Pull request by gaben3722**
## Description

This Pr fixes the Twitter Bot reply timer issue. It will now run on a four minute timer. 

Fixes # (Bot Timer)

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

https://www.loom.com/share/4e719df15da44b1f9187462f8ce66cd6

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -194,7 +190,8 @@ async def to_approve():
     return needs_approval
 
 
-@repeat_every(seconds=60 * 60)
+@app.on_event("startup")
+@repeat_every(seconds=60 * 4)
 @app.get("/advance-all/")
```
Comment (ChristopherKchilton) [inline app/main.py:196]: This seems to be an issue with Fast api and it does not recognize a timer without a start up event. This should fix the bug.

**Review by ChristopherKchilton — APPROVED** [review]
The only concern would be how often the twitter bot should auto update. This is something that could be of concern. Anything too fast will be inefficient since tweepy has some lag time and anything too slow will cause people to not interact in conversations with the twitter bot.

File: app/main.py [hunk 2]
```diff
@@ -194,7 +190,8 @@ async def to_approve():
     return needs_approval
 
 
-@repeat_every(seconds=60 * 60)
+@app.on_event("startup")
+@repeat_every(seconds=60 * 4)
```
Comment (iknight7000) [inline app/main.py:195]: I can see this is where you updated the timer to run every 4 minutes.

**Review by iknight7000 — APPROVED** [review]
The timing on the bot definitely needed to be approved. Thank you for this.

File: app/main.py (hunk 2 above)
Comment (BrokenShell) [reply app/main.py:195]: lets remove the API call after this in a future PR.

**Review by BrokenShell — APPROVED** [review]

**Pull request closed without merging**